src/main/java/uk/co/downthewire/jLTE/simulator/FadingData.java

```diff
@@ -47,19 +52,22 @@ public void readFading(int iteration) throws IOException {
         int modifiedIteration = iteration % 120;
```

Comment:
If I recall this was for those cases that we wanted to run the simulator longer than 120 iterations. In this case we reuse the fading data. This is because the fading data is expensive to compute and store (depending on your resources) so we choose to reuse it every 120 iterations. This allowed us to run longer simulations without the extra cost that more fading data would incur.